pkg/pb/contacts.override.gorm.go

```diff
@@ -11,17 +12,42 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-// BeforeToORM will add the primary e-mail to the list of e-mails if it isn't
+// AfterToORM will add the primary e-mail to the list of e-mails if it isn't
 // present already
-func (m *Contact) BeforeToORM(ctx context.Context, c *ContactORM) error {
-	if m.PrimaryEmail != "" {
-		for _, mail := range m.Emails {
-			if mail.Address == m.PrimaryEmail {
-				return nil
-			}
+func (m *Contact) AfterToORM(ctx context.Context, c *ContactORM) error {
+	if m.PrimaryEmail == "" {
+		return nil
+	}
+
+	var primary *EmailORM
+
+	emails := []*EmailORM{}
+	for _, e := range c.Emails {
+		if e.Address != m.PrimaryEmail {
+			e.IsPrimary = new(bool)
+			*e.IsPrimary = false
+			emails = append(emails, e)
+		} else {
+			e.IsPrimary = new(bool)
+			*e.IsPrimary = true
+			primary = e
+		}
+	}
+
+	if primary == nil {
+		accountID, err := auth.GetAccountID(ctx, nil)
```

Comment:
I think it's not very good that app code deals with `AccountID` directly, could you create a PB Email, convert it to ORM and then set `IsPrimary` to `true`?